config/settings_schema.json

```diff
@@ -7,6 +7,32 @@
     "theme_documentation_url": "https://help.shopify.com/manual/online-store/themes",
```

Comment:
Is this expected? (when it's at the largest size)

https://screenshot.click/08-49-udirq-zfrjx.mp4